build.gradle.kts

```diff
@@ -1,60 +1,17 @@
-import org.hypertrace.gradle.publishing.License.APACHE_2_0
+import org.hypertrace.gradle.publishing.HypertracePublishExtension
+import org.hypertrace.gradle.publishing.License
 
 plugins {
-  `java-gradle-plugin`
-  id("org.hypertrace.publish-plugin") version "0.3.0"
-  id("org.hypertrace.ci-utils-plugin") version "0.1.1"
+    id("org.hypertrace.ci-utils-plugin") version "0.2.0"
+    id("org.hypertrace.publish-plugin") version "0.3.0" apply false
+    id("org.hypertrace.repository-plugin") version "0.2.3"
 }
 
-group = "org.hypertrace.gradle.publishing"
-
-java {
-  targetCompatibility = JavaVersion.VERSION_1_8
-  sourceCompatibility = JavaVersion.VERSION_1_8
-}
-
-repositories {
-  jcenter()
-}
-
-gradlePlugin {
-  plugins {
-    create("gradlePlugin") {
-      id = "org.hypertrace.publish-plugin"
-      implementationClass = "org.hypertrace.gradle.publishing.PublishPlugin"
-    }
-  }
-}
-
-dependencies {
-  // Do not upgrade without reviewing. See PublishPlugin.createModuleMetadataPublication
-  implementation("com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.5")
-
-  constraints {
-    implementation("org.apache.httpcomponents:httpclient:4.5.12") {
-      because("Multiple vulnerabilities in versions included via bintray plugin")
+subprojects {
```

Comment:
The repo is converted to a multi-projects repo and a new subproject `hypertrace-gradle-publish-maven-central-plugin` is added.